code/datums/mind.dm

```diff
@@ -41,6 +41,7 @@
 	var/assigned_role
 	var/special_role
 	var/list/restricted_roles = list()
+	var/list/datum/objective/objectives = list()
```

Comment:
uh why

Reply:
Not even sure this one applies anymore it might be remnants from the last objective change

Reply:
line 307-309 in cryopod.dm. I think this is cutting their objectives from the roundend report